pages/index.js

```diff
@@ -1,20 +1,19 @@
 import React from 'react';
 import Layout from '../components/Layout';
 import EventList from '../components/EventList';
-import EVENTS from '../fixtures/events'
+import { fixtureApi } from '../services';
```

Comment:
I'm not clean on what a fixture is -- I assumed it was an OSDI-related word for event but it sounds like it's something else